nibabel/nifti1.py

```diff
@@ -723,6 +723,11 @@ def set_data_shape(self, shape):
         If ``ndims == len(shape)`` then we set zooms for dimensions higher than
         ``ndims`` to 1.0
 
+        Nifti1 images can have up to seven dimensions. For Nifti surface files,
```

Comment:
When you say "Nifti surface files" - do you mean that this is part of the NIfTI standard somewhere?  Or you just mean NIfTI as creatively reinterpreted by Freesurfer?